collections-csharp/ConcurrentQueueInCSharp/ConcurrentQueueInCSharp.Console/Producer.cs

```diff
@@ -0,0 +1,27 @@
+﻿using ConcurrentQueueInCSharp.Models;
+
+namespace ConcurrentQueueInCSharp
+{
+    public class Producer
+    {
+        private readonly OrderMessageBus _messageBus;
+        private readonly int _numberOfMessages;
+
+        public Producer(OrderMessageBus messageBus, int numberOfMessages)
+        {
+            _messageBus = messageBus;
+            _numberOfMessages = numberOfMessages;
+        }
+
+        public Task Produce()
+        {
+            return Task.Factory.StartNew(() =>
```

Comment:
We can use `Task.Run` to shorten the line. It is also the more standard practice.

Reply:
Good point.